src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/fixtures/UserFixture.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.manageusersapi.fixtures
+
+import java.time.LocalDateTime
+import java.util.UUID
+import uk.gov.justice.digital.hmpps.manageusersapi.resource.external.ExternalUserDetailsDto
+
+class UserFixture {
+
+    fun createExternalUserDetails(userId: UUID, username: String, email: String): ExternalUserDetailsDto {
+        return ExternalUserDetailsDto(
+            userId = userId,
+            username = username,
+            email = email,
```

Comment:
I would prob put defaults on these and make the fields optional.

Reply:
Done